src/main/java/redis/clients/jedis/UserACL.java

```diff
@@ -0,0 +1,53 @@
+package redis.clients.jedis;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class UserACL {
```

Comment:
@gkorland To me, `UserACL` seems more like User's ACL. `ACLUser` may be better to represent ACL's User. Or, perhaps even better `AccessControlUser`. WDYT?

Reply:
Used AccessControlUser